pallets/loans/src/entities/loans.rs

```diff
@@ -533,31 +531,42 @@ pub struct ActiveLoanInfo<T: Config> {
 
 	/// Current outstanding interest of this loan
 	pub outstanding_interest: T::Balance,
+
+	/// Current price for external loans
+	/// - If oracle set, then the price is the one coming from the oracle,
+	/// - If not set, then the price is a linear accrual using the latest
+	///   settlement price.
+	/// See [`ExternalActivePricing::current_price()`]
+	pub current_price: Option<T::Balance>,
```

Comment:
I think we could use the valuation method of the internal pricing here too. Removing the Option. ANd I would name it current_value.
 

cc: @denniswell

Reply:
Valuation method is just the present value for internal pricing loans. So we should do the same for external pricing loans too if go with this change. 

This is related to my question:

> Should we modify present_value() to now use current_price instead of latest_settlement_price?

For external pricing loans `present_value = quantity * latest_settlemente_price`. So maybe we do not need to use another field here and just update how `present_value` is computed using now `current_price` instead of `latest_settlement_price`

Reply:
Ah sorry, my bad! I think that is the best idea!

Reply:
Maybe we still want to put `current_price` here to allow the UI just to draw the price. Otherwise, they should divide the present_value / outstanding_quantity to obtain it.

Reply:
I think that is confusing more ATM. The total value is anyways of greatest interest.

Reply:
Ok, then we should document that to draw the current price used for a loan, the UI should do: `present_value / outstanding_quantity` for cases were oracles are not set instead of using directly `latest_settlement_price` as before.

Reply:
cc @onnovisser

Reply:
I just realized that `present_value` is always affected by how much the loan has been written down. Maybe it's quite difficult to extract the "current price" for Apps.

Reply:
I think all that is relevant mostly is the value anyways.

Reply:
Regarding this topic, I would say that if Apps needs to draw the price, leave the `current_price` as it is. Otherwise, if they need it and we do not offer it by the RuntimeAPI, then they need to perform not-trivial computations, and future changes to present_value will break that.

@onnovisser are you drawing right now the `settlementPrice` of an external loan?

Reply:
@lemunozm can we remove that one here?

Reply:
Can we decide after getting feedback from https://kflabs.slack.com/archives/C04GARZ532T/p1709024010720729 ?

Reply:
Onno is out, which is who is taking care of it, I think, so probably we do not have feedback soon regarding this.

Seeing the apps code, they need to fetch the price as `current_price` now to fulfill this part: https://github.com/centrifuge/apps/blob/beea676a936a14597f270e931387a4df1ce7ca0d/centrifuge-app/src/utils/getLatestPrice.ts#L10 which is later used to draw in the loan page. 

I would say we need to keep this here. Do you have any argument against this, @mustermeiszer? I do not see you very in favor of adding it.